director/origin_api.go

```diff
@@ -75,7 +75,7 @@ func CreateAdvertiseToken(namespace string) (string, error) {
 	}
 
 	tok, err := jwt.NewBuilder().
-		Claim("scope", "pelican.advertise").
+		Claim("scope", utils.Pelican_Advertise.String()).
```

Comment:
Let's add a helper function here as well.  Maybe call it `Scope` for adding a single scope to the token and `Scopes` for a slice of `TokenScope` objects?

Reply:
This would change the syntax here to:

```
	tok, err := jwt.NewBuilder().
		Scope(utils.Pelican_Advertise).
		...
```

Reply:
The other nice thing about creating a helper function here would be the addition of type safety -- it'll be easy for reviewers to spot mistakes (as we should never have calls of `Claims("scope", ...)`).

Reply:
This should be in the scope for an additional work #559 to replace all existing token creation logic by `utils.CreateToken` function. In this PR, I will add scope type enforcement but won't touch the existing token creation logic, so that we won't repeat ourselves in #559.